src/pages/_includes/ui/select.html

```diff
@@ -56,7 +65,7 @@
 		var el;
 		window.TomSelect && ({% if jekyll.environment == 'development' %}window.tabler_select["select-{{ id }}"] = {% endif %}new TomSelect(el = document.getElementById('select-{{ id }}'), {
 			copyClassesToDropdown: false,
-			dropdownClass: 'dropdown-menu',
+			dropdownClass: 'dropdown-menu ts-dropdown',
```

Comment:
by adding this class, [the original tomselect rules](https://github.com/orchidjs/tom-select/blob/master/src/scss/tom-select.bootstrap5.scss#L76) now will be applied